pyphare/pyphare/pharesee/particles.py

```diff
@@ -0,0 +1,40 @@
+
+import numpy as np
+
+class Particles():
+    def __init__(self, **kwargs):
+        if "box" in kwargs:
+            box = kwargs["box"]
+            self.iCells = np.random.randint(box.lower, high=box.upper+1, size=box.size()*100)
+            self.deltas = np.random.rand(box.size()*100)
+            self.vx = np.random.randn(box.size()*100)
+            self.vz = np.random.randn(box.size()*100)
+            self.vy = np.random.randn(box.size()*100)
+        else:
+            self.iCells = kwargs["icell"]
+            self.deltas = kwargs["deltas"]
+            self.vx = kwargs["vx"]
+            self.vy = kwargs["vy"]
+            self.vz = kwargs["vz"]
+
+    def select(self, box):
+        idx = np.where((self.iCells >= box.lower) & (self.iCells <= box.upper))[0]
```

Comment:
assumption, box has AMR indexes of the same level as the data that the current instance is created from

Reply:
sure. but I don't really see how to assert that...

Reply:
a comment is enough

Reply:
done